sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -301,9 +298,6 @@ case class AlterTableSerDePropertiesCommand(
     "ALTER TABLE attempted to set neither serde class name nor serde properties")
 
   override def run(sparkSession: SparkSession): Seq[Row] = {
-    DDLUtils.verifyTableProperties(
```

Comment:
We will let users change data source options. But, the command will not be `serdeproperties` since it is hive specific.